src/EmployerAan/SFA.DAS.EmployerAan.Api/Controllers/MembersController.cs

```diff
@@ -23,4 +29,35 @@ public async Task<IActionResult> GetMembers([FromQuery] GetMembersRequestModel r
         var response = await _mediator.Send((GetMembersQuery)requestModel, cancellationToken);
         return Ok(response);
     }
+
+    [HttpPut("{memberId}")]
+    [ProducesResponseType(StatusCodes.Status204NoContent)]
+    public async Task<IActionResult> UpdateMemberProfileAndPreferences(
```

Comment:
Get and Update of profile and preferences should have stayed in the MemberProfilesController.

Reply:
Don't bother changing this though as I can see similar implementation in App outer has been approved.

Reply:
Acutally method "UpdateMemberProfileAndPreferences" is updating three tables member,memberprofile and memberpreference.So i checked apprentice outer api for this and found that we placed "UpdateMemberProfileAndPreferences" method in member controller so i followed the same pattern in employer outer as it is also updating the member table.